ubuntudesign/partners.ubuntu.com
# add whitepace to public-cloud partnerss include

## Done

*added whitespace to make _logo-list.html read the query key pair

## QA

1. go to /programmes/public-cloud
2. see that the logo cloud has the correct logos
3. see that the link 'See all public cloud partners ›' has the get parameters it should /find-a-partner?programme=certified-public-cloud

## Issue / Card

Fixes [lp bug 1505395](https://bugs.launchpad.net/ubuntu-partner-website/+bug/1505395)
